zeebe/gateway/src/main/java/io/camunda/zeebe/gateway/impl/broker/request/BrokerDeployResourceRequest.java

```diff
@@ -23,9 +23,8 @@ public BrokerDeployResourceRequest() {
     setPartitionId(Protocol.DEPLOYMENT_PARTITION);
   }
 
-  public BrokerDeployResourceRequest addResource(final byte[] resource, final String resourceName) {
+  public BrokerDeployResourceRequest addResource(final String resourceName, final byte[] resource) {
```

Comment:
👍 I really like this ordering! Makes sense and makes things more readable (at least to me 😄).